docs/_dg/Implementation.md

```diff
@@ -13,6 +13,40 @@ The `tag` feature tags an item with with the provided tag name in FoodRem. If bo
 6. If Step 5 completes without any exceptions, a new copy of the item is created and the tag would be added to that item. This new copy of item would replace the original item in the database.
 
 #### Why it is implemented this way
-When the `TagCommand` is executed to tag a tag to an item, a copy of the item is created and the tag is added to it before replacing this new copy of the item with the original item in the list of items in FoodRem. We chose to replace the original item with the new item because this will allow the UI to detect a change in the `UniqueItemList` and thus update and show the item with their new tag included. 
+When the `TagCommand` is executed to tag a tag to an item, a copy of the item is created and the tag is added to it before replacing this new copy of the item with the original item in the list of items in FoodRem. We chose to replace the original item with the new item because this will allow the UI to detect a change in the `UniqueItemList` and thus update and show the item with their new tag included.
 
-![TagSequenceDiagram](../images/TagSequenceDiagram.png)
+![TagSequenceDiagram](images/TagSequenceDiagram.png)
+
+### Sorting Items
+
+#### Implementation
+The `sort` feature sorts the list of items currently displayed to the user by specified flag(s). It is possible to sort by one or more criteria. A list of available criteria for sorting includes:
+
+* Name
+* Quantity
+* Type
+* Quantity
+* Bought Date
+* Expiry Date
+
+Currently, sorting is performed ascendingly. A future implementation will allow sorting either ascendingly or
+descendingly.
+
+#### How the `sort` command works
+
+1. The user can perform other commands that modifies the presented item list, such as the `find` command.
+2. The user specifies one or more criteria used for sorting. Available criteria for sorting are listed above.
+3. If the displayed item list is empty, then an empty list is displayed.
+4. If the item list is not empty, the items are sorted according to the specified criteria. Sorting is performed in order of the criteria specified.
+
+![TagSequenceDiagram](images/SortItemsSequenceDiagram.png)
+
+#### Why is it implemented this way
+There is a `SortedList` obtained from an immutable item
```

Comment:
Just some style issue:
For this paragraph, can I check why are there line breaks instead of having the whole paragraph in a line (unless you are starting a new paragraph/point)? I believe the 120 char limit only applies to code but not md text right?

Reply:
Ah right. I disabled formatting for `.md` files in my IntelliJ because of linting issues, so I used `\n` just in-case of any word limit. You're right: I didn't have to do it. 
![image](https://user-images.githubusercontent.com/53888954/196588276-e972c532-ec4c-4786-a6ec-6ff8b8a69323.png)